src/components/ChatApp/Settings/ShareOnSocialMedia.js

```diff
@@ -95,7 +95,7 @@ const ShareOnSocialMedia = props => {
               }
             />
           </div>
-          <div style={props.headingStyle}>
+          {/* <div style={props.headingStyle}>
```

Comment:
Why comment it when you can remove it? :wink:

Reply:
@AakashMallik I deleted the part as suggested by you, it would be great if you review it once again..Thanks..Actually I was trying to see if the code ran on local host perfectly by commenting and then filed a PR for the same :))  
Now made the requested changes.